examples/tv-app/linux/include/wake-on-lan/WakeOnLanManager.cpp

```diff
@@ -45,11 +47,13 @@ CHIP_ERROR WakeOnLanManager::Init()
 
 void WakeOnLanManager::store(chip::EndpointId endpoint, char macAddress[18])
 {
-    char addressBuffer[19];
-    snprintf(addressBuffer, 19, "%c%s", 18, macAddress);
+    char cString[18];
+    strcpy(cString, macAddress);
```

Comment:
This part does not make sense to me. Is the incoming thing not null-terminated?  If so, how does strcpy make sense?  And if it is, why do we need to copy it at all?  Just because the template thing doesn't realize `macAddress` is the right type due to type decay?  But then the solution is to fix the template, I'd think....

In any case, unchecked strcpy is not ok; we should always be using `strncpy` or the even safer `chip::Platform::CopyString` (aka `strlcpy`).

Reply:
I've changed the template for `ZclString/MakeZclCharString`